tests/test_models.py

```diff
@@ -8,18 +8,35 @@
 Tests for `django-user-verification` models module.
```

Comment:
Should I call it `test_services`?  Or should I specify in the test file that I'm making tests for `services` models module?